src/evaluations/report_generator.py

```diff
@@ -140,27 +140,29 @@ def add_parsed_sketch_estimator_name_cols(cls, df, sketch_estimator_name_col):
     return pd.concat([df, names_df], axis=1)
 
   @classmethod
-  def widen_num_estimable_sets_df(cls, df):
+  def widen_num_estimable_sets_df(cls, df, error_metric_column):
     """Widen the number of estimable sets with statistics df.
 
     Args:
       df: a data frame that contains the number of estimable sets by estimators
-        and scenarios, and the relative error stats at the number of estimable
+        and scenarios, and the error metric stats at the number of estimable
         sets.
+      error_metric_column: name of column containing the error metric to be
+        used for widening.
 
     Returns:
       A wide format of the number of estimable sets and the relative error
       stats.
     """
-    def _format_num_estimable_sets_df(df):
+    def _format_num_estimable_sets_df(df, error_metric_column):
       df = df.copy()
       df['num_estimable_sets_cell'] = (
           df[analyzer.NUM_ESTIMABLE_SETS].astype('str')
           + '<br>relative_error: mean='
-          + df[simulator.RELATIVE_ERROR_BASENAME + '1_mean'].round(
+          + df[error_metric_column + '_mean'].round(
```

Comment:
curious why do we need appended `1` for reach, but not for frequency?

Reply:
We now compute relative error by frequency, so we have relative_error_1, relative_error_2, ..., where relative_error_k represents the estimation error for frequency k or more.  Thus, in this new definition, relative_error_1 has the same meaning as relative_error had in the previous version of the code.  For frequency, we use the shuffle_distance metric rather than the relative_error metric.  Shuffle_distance is something like the average relative error over all frequencies (up to some maximum frequency).  So, it is not frequency-specific.  I hope this explanation helps!